### PR TITLE
010 role detail

### PR DESCRIPTION
**Pull request by anne-moore**

### CODE REVIEW

File: src/services/JobRoleService.ts [hunk 1]
```diff
@@ -14,3 +15,22 @@ export const getJobRoles = async (): Promise<JobRolesResponse[]> => {
...
     }
 }
+
+export const getJobRoleById = async (id: string): Promise<JobRole> => {
+    try {
+        const response: AxiosResponse = await axios.get(URL + id);
+
+        if(response.data.status != "open") {
+            return null;
+        }
```
Comment (shaunganley) [inline src/services/JobRoleService.ts]: the acceptance criteria don't ask for this so I would remove it (we'll want to be able to view closed job role information in the future

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: src/services/JobRoleService.ts (hunk 1 above)
Comment (anne-moore) [reply src/services/JobRoleService.ts]: Removed

**Review by shaunganley — APPROVED** [review]

**Review by joel-and — APPROVED** [review]

File: views/jobRoleDetail.html [hunk 2]
```diff
@@ -34,8 +34,8 @@ <h4 id="responsibilities"> Responsibilities</h4>
                     <p id="responsibilities">{{ jobRole.responsibilities }}</p>
 
                 
-                    <p id="capability"><b>Capability: </b>{{ jobRole.capability }}</p>
-                    <p id="ban"><b>Band: </b>{{ jobRole.band }}</p>
+                    <p id="capability"><b>Capability: </b>{{ jobRole.capabilityID }}</p>
+                    <p id="ban"><b>Band: </b>{{ jobRole.bandID }}</p>
```
Comment (shaunganley) [inline views/jobRoleDetail.html]: you shouldn't be showing IDs on the screen, instead get back the band and capability names and display here

**Review by shaunganley — CHANGES_REQUESTED** [review]
this last change isn't required. instead you should update the API to return the names rather than the IDs

File: src/models/JobRole.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export type JobRole = {
```
Comment (shaunganley) [inline src/models/JobRole.ts]: is this needed?

**Review by shaunganley — APPROVED** [review]

File: src/models/JobRole.ts (hunk 3 above)
Comment (anne-moore) [reply src/models/JobRole.ts]: Removed and using JobRoleResponse instead

**Pull request merged**